bindings/c/foundationdb/fdb_c.h

```diff
@@ -115,8 +115,6 @@ typedef struct keyvalue {
 } FDBKeyValue;
 #endif
 
-#pragma pack(pop)
```

Comment:
is this intended? this might affect the memory layout for `mappedkeyvalue`, so earlier versions of FDB could be different w.r.t. the layout?

Reply:
I am concerned about the case that 7.3 client uses multi-version client, which uses 7.1 C binding (compiled early) can talk with 7.1 cluster.